tool/jt.rb

```diff
@@ -529,6 +529,13 @@ def build(*options)
   def build_truffleruby(*options, sforceimports: true)
     mx 'sforceimports' if sforceimports
 
+    env_path = "#{TRUFFLERUBY_DIR}/mx.truffleruby/env"
+    env_content = File.exist?(env_path) ? File.read(env_path) : ''
+    dynamic_import = 'DEFAULT_DYNAMIC_IMPORTS=/tools'
+    unless env_content.include? dynamic_import
+      File.write(env_path, env_content + "#{dynamic_import}\n")
```

Comment:
If I'm reading this correctly, this code assumes an existing env file has a trailing newline character. Otherwise, the append is going to write out invalid content. I think you should prepend a newline to `dynamic_import` if `env_content` doesn't end with one.

Reply:
An easy way could be to add the new dynamic_import first, and the existing content after.